src/url-loader/prefixed-url-loader.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ * @license
+ * Copyright (c) 2015 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at
+ * http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at
+ * http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+import {UrlLoader} from './url-loader';
+
+/**
+ * Resolves requests via a given delegate loader for URLs matching a given
+ * prefix.  URLs are provided to their delegate without the prefix.
```

Comment:
I am old and still I use two spaces after a period everywhere because it actually is better but the whole world changed when proportional fonts came around and people have since been learning this _wrong_.

I realize I am fighting a losing battle and even our codebase predominantly uses single space after end-of-sentence punctuation-- even files specifically intended to be rendered with monospace fonts, which has typically been the last safe refuge of my two-space kind.

I will convert this two-space to a single-space, but I do it with a heavy heart and only in the knowledge that some day the truth will be known that single-space after the end of sentence punctuation was the wrong direction for a world, though this wisdom may be rediscovered too late.

![too-late](https://cloud.githubusercontent.com/assets/578/25838247/0b236f4e-3447-11e7-9b8a-49eb2fc7475c.jpg)